amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/model/retentionschedule/RetentionSchedule.java

```diff
@@ -46,4 +46,13 @@ public class RetentionSchedule extends TestModel
     private boolean isRecordLevel;
     private boolean isUnpublishedUpdates;
     private List<RetentionScheduleActionDefinition> actions;
+
+    public boolean getIsRecordLevel()
```

Comment:
Is it possible to use lombok getter and setter instead creating the new get() and set() methods ?

Reply:
yes we are using lombok, but for this boolean property we are facing issue while calling api so we need to use getter for this field.